.github/workflows/validate-new-issue.yml

```diff
@@ -27,15 +27,23 @@ jobs:
         # - order doesn't matter
         # - case-sensitive tho
         function extract-headings { { grep '^#' "$1" || echo -n ''; } | sort; }
-        extract-headings <(jq -r .issue.body "$GITHUB_EVENT_PATH") > headings-in-issue
+        jq -r .issue.body "$GITHUB_EVENT_PATH" > issue
+        extract-headings <(cat issue) > headings-in-issue
         for template in $(ls .github/ISSUE_TEMPLATE/*.md 2> /dev/null); do
+          # Strip front matter. https://stackoverflow.com/a/29292490/14946704
+          sed -i'' '1{/^---$/!q;};1,/^---$/d' "$template"
           extract-headings "$template" > headings-in-template
           echo -n "$(basename $template)? "
           if [ ! -s headings-in-template ]; then
             echo "No headers in template. 🤷"
           elif [ -z "$(comm -23 headings-in-template headings-in-issue)" ]; then
             echo "Match! 👍 💃"
-            exit 0
+            if diff -Bw "$template" issue > /dev/null; then
```

Comment:
Guessing `-Bw` is to detect whitespace-only changes?

Reply:
Yup. I'll add a comment.